tests/sentry/utils/performance_issues/test_performance_detection.py

```diff
@@ -878,18 +878,28 @@ def test_detects_slow_span_but_not_n_plus_one_in_query_waterfall(self):
 
         _detect_performance_problems(query_waterfall_event, sdk_span_mock)
 
-        assert sdk_span_mock.containing_transaction.set_tag.call_count == 4
+        assert sdk_span_mock.containing_transaction.set_tag.call_count == 6
```

Comment:
The event actually detects a consecutive db issue and a N+1 issue, because were running in detect only, this is fine but we'll need to make sure we don't create two issues.

Reply:
Yeah we should be able to search for cases where `_pi_consecutive_db` is tagged but `_pi_n_plus_one_db_query` is not tagged to test this in prod before sorting this out (could happen before we return issues from the detector, for example). I'm okay with punting on this while we collect data.